package.json

```diff
@@ -19,8 +19,12 @@
   },
   "dependencies": {
     "@auth0/auth0-vue": "^2.2.0",
+<<<<<<< HEAD
```

Comment:
Seems like you have not resolved all the merge-conflicts 😅

Reply:
Yes, i have to resolve it.